normal/matrix.py

```diff
@@ -0,0 +1,107 @@
+def __get_cols_number(matrix):
+    return len(matrix[0])
+
+
+def __get_rows_number(matrix):
+    return len(matrix)
+
+
+def __get_nth_row(matrix, n):
+    return matrix[n]
+
+
+def __get_nth_col(matrix, n):
+    col = []
+    for row in matrix:
+        col.append(row[n])
+    return col
+
+
+def __init_result_matrix_of_size(rows_num, cols_num):
+    matrix = []
```

Comment:
use numpy array instead of mutable python list of list. Its more efficient and clearer solution

Reply:
i'll implement it in next version.